Changelog.md

```diff
@@ -2,6 +2,10 @@
 
 ## Upcoming Release
```

Comment:
Any documentation that we need to update explaining the new feature?

Reply:
Hmm... I'll have to think about that. My first instinct is "no" since we don't document every feature.

Reply:
I was wrong. The features are listed out in the readme. So I updated them to include settings and the backup features.